docs/ml-guide.md

```diff
@@ -868,6 +868,132 @@ jsc.stop();
 
 </div>
 
+## Example: Model Selection via Train Validation Split
+In addition to  `CrossValidator` Spark also offers
+[`TrainValidationSplit`](api/scala/index.html#org.apache.spark.ml.tuning.TrainValidationSplit) for hyper-parameter tuning.
+`TrainValidationSplit` only evaluates each combination of parameters once as opposed to k times in
+ case of `CrossValidator`. It is therefore less expensive, but will not produce as reliable results.
+
+`TrainValidationSplit` takes an `Estimator`, a set of `ParamMap`s provided in the `estimatorParamMaps` parameter, and an
+[`Evaluator`](api/scala/index.html#org.apache.spark.ml.Evaluator).
```

Comment:
@jkbradley has told me in the past to keep the scala API links under the scala codetab and to create a separate version linking java API docs under java codetabs.

However, it looks like we have been inconsistent throughout `ml-guide.md` about this so I'm OK with how it looks now. Just wanted to flag it in case others have something to say.